### PR TITLE
ELY-2643 : Simplify assertions in testIterationAsStringValue method in  MaskCommandTest

### PR DESCRIPTION
**Pull request by nidhiazad**
https://issues.redhat.com/browse/ELY-2643

Run the test case using mvn clean install -Dtest=MaskCommandTest

### CODE REVIEW

File: tool/src/test/java/org/wildfly/security/tool/MaskCommandTest.java [hunk 1]
```diff
@@ -126,12 +126,13 @@ public void testIterationAsStringValue() {
 
         String[] args = { "--secret", secret, "--salt", salt, "--iteration", "abcd" };
 
+
```
Comment (ashley-abdelsayed98) [inline tool/src/test/java/org/wildfly/security/tool/MaskCommandTest.java:129]: nit: extra line

**Review by ashley-abdelsayed98 — APPROVED** [review]
LGTM, thanks!

**Review by PrarthonaPaul — APPROVED** [review]
Looks good to me!

File: tool/src/test/java/org/wildfly/security/tool/MaskCommandTest.java (hunk 1 above)
Comment (skyllarr) [reply tool/src/test/java/org/wildfly/security/tool/MaskCommandTest.java:129]: yes just a minor @nidhiazad, there is an extra line here. But LGTM, I will merge this one. Thank you @nidhiazad !

**Pull request merged**